tests/tasks/assert_profile_present.yml

```diff
@@ -10,3 +10,8 @@
   assert:
     that: lsr_net_profile_ansible_managed
     msg: "profile {{ profile }} does not have the ansible managed comment"
+
+- name: "assert that fingerprint comment in '{{ profile }}' is present"
```

Comment:
Ansible lint 6.x requires starting all task names with an uppercase letter.   

"Assert that... "